### DISCUSSION TITLE
User Feedback Session - 2024-09-11

### DISCUSSION THREAD

**Discussion by jwildfire** [Gilead-BioStats/gsm.app#223] [General]
Will add comments capturing user feedback/requests

**Comment by lauramaxwell** [comment]
These are notes from the user feedback session on 9/11/24

## General Feedback
- Typical use is to have ADO or report open on one screen and have this app on the other screen, so might not have a lot of use for the Study Overview page
  - Not sure if this is how _everyone_ uses the app, so probably worth keeping as the landing page.
- This participant details has been very helpful for ASCENT-5 and other studies that have used this

## Updates / Bugs
- Cross-linking clicks between tabs (issues #196 and #197). (Jenn)
- Make sure that column sorts are successive, so new column sort retains the sort of the first column sort. (Gosia)
- Potential issue with `Reset All` button- disconnects from server
- Adjustable column widths in Participant Domain table on Participant Details page
  - This might not be necessary with new spec process, but worth looking into.
- Navigate from the Participant Details page to the Metric Details page when metric selected on the sidebar is changed
  -  Jon has implemented this in {gsm.app} v2
- Ensure Site Metadata table is accurate- v1 has some questionable values, leading them to believe it is not being populated correctly

## New Features
- Want a filter on all **open** queries (Sathya) and other features as well
  - can currently sort, but that may not be sufficient because they want to sort on multiple columns at once (Gosia)
  - maybe use Teal?
- Show all participants (with or without a KRI) from a site in a view
  - Drill down at the site-level
- Export a csv?
  - May not be permitted because of data sensitivity- this app might be Admin-only
  - Another solution- make a full-screen view of the table or other UX improvements within the app
- Logout button? 
  - this would likely need to be implemented in Gismo
- Subject Status as a column in all Site Participants tables
 
## Longer Term Feature Requests
- Participant profile tab or updated visuals for participant tab
- Site profile tab (Sathya/Jenn)
- Visualize the underlying raw data in addition to or instead of tables (Randall)
  - Will likely be handled in the AE Deep Dive app/tab/module
